distribution/kpkginstall/runtest.sh

```diff
@@ -38,14 +44,29 @@ function set_package_name()
       PACKAGE_NAME=kernel
   fi
 
+  # Append "-debug" if we were asked to install the debug kernel.
+  if [[ "${KPKG_INSTALL_DEBUG:-}" == "yes" ]]; then
```

Comment:
Hm, why the `:-`? if you're not providing any default, isn't it the same to write the following?
```
if [[ "$KPKG_INSTALL_DEBUG" == "yes" ]]
```